nuke_frontend/src/styles/user-profile.css

```
@@ -56,6 +56,22 @@
+     (--bg / --surface / --text / --border), which resolve dark in the app's
+     dark theme — producing random dark slabs amid the light record. Remap the
+     global tokens to the profile's own light palette for the subtree so every
+     surface is the same light treatment. No new colors: aliases of --up-*. */
+  --bg: var(--up-bg);
+  --surface: var(--up-surface);
+  --surface-hover: var(--up-row-alt);
+  --surface-elevated: var(--up-surface);
+  --text: var(--up-ink);
+  --text-muted: var(--up-data-dim);
+  --border: var(--up-ghost);
+  color-scheme: light;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)